DummyData/dummyData.js

```diff
@@ -70,6 +105,11 @@ const songData = [
     fav_count: 69,
     path_to_artwork: 'https://i.imgur.com/AD3MbBi.jpeg',
     user_id: 'ClawzOut',
+    tags : {
+      id: "7b987c5b-6abe-4821-9315-99fc61781eb1",
+      song_id : "d76d275d-0c54-4f54-8627-edf27712195c",
+      name: "tag"
+    }
```

Comment:
I mentioned this below in the models file, but I think we can just have tags be an array of tag names....I can't think of when we would need to use the tag_id, and the song_id is already present in the songData as 'id'

Reply:
I agree, that makes sense! The original model function "getAllSongs"/"getSongsByUser" that was already merged was written by someone else. I wasn't sure whoever wrote the original query if they needed it or not so I left the query as it was.